api/query/cache/index/index.go

```diff
@@ -112,21 +117,30 @@ func (i *shardedWPTIndex) IngestRun(r shared.TestRun) error {
 	// top-level test (i.e., not subtests) integral ID of each test in the report.
 	//
 	// Create RunResults for each shard's partition of this run's results.
-	numShards := uint64(len(i.shards))
-	runResults := make([]RunResults, numShards)
-	for j := uint64(0); j < numShards; j++ {
-		runResults[j] = NewRunResults()
+	numShards := len(i.shards)
```

Comment:
Will the length of `i.shards` change? (I don't see anything wrong, but just want to confirm.)

Reply:
The design (which I don't think is violated by any of the code) is that the number of shards is fixed at creation time.